### PR TITLE
docs: v0.8 release notes

### PR DESCRIPTION
**Pull request by KwilLuke**

### CODE REVIEW

File: docs/release-notes/release-v0.8.md [hunk 1]
```diff
@@ -0,0 +1,302 @@
...
+
+### SDK (`core` module)
+
+This release of Kwil DB uses version 0.2.1 of the `core` Go module.
+
+A Go application that supports Kwil DB v0.8.0 should be developed using the `core` module specified in the go.mod as follows:
+
+```go
+require github.com/kwilteam/kwil-db/core v0.2.1
+require github.com/kwilteam/kwil-db/parse v0.2.1 // to parse Kuneiform files
```
Comment (jchappelow) [inline docs/release-notes/release-v0.8.md:181]: Mmmm,  I should have done a section like this for 0.9.  I may circle back on that.

File: docs/release-notes/release-v0.8.md [hunk 2]
```diff
@@ -0,0 +1,302 @@
...
+```go
+type Signature struct {
+    Signature []byte `json:"sig"` // was `json:"signature_bytes"`
+    Type string `json:"type"` // was `json:"signature_type"`
+}
+```
+
+#### `core/types`
+
+- The `Schema` struct and all of it's composing types are now mirrored in `core/types.Schema`.
```
Comment (jchappelow) [inline docs/release-notes/release-v0.8.md:203]: it's => its

That's a jchappelow typo, I'm sure

**Review by jchappelow — APPROVED** [review]

**Comment by Yaiba** [comment]
should backport to release-v0.8?

**Comment by jchappelow** [comment]
> should backport to release-v0.8?

I personally don't think so, since people typically read doc files from the default branch (`main`)

**Pull request merged**